### DISCUSSION TITLE
Permission Denied after installing Dbus-serialbattery

### DISCUSSION THREAD

**Discussion by Martinmaker92** [Louisvdw/dbus-serialbattery#843] [Q&A]
Hey guys i´ve used an old version of ther serial battery monitor on my Victron 3 Phase system with 4 battery packs with JK BMS.

it worked fine for a year until friday. I tried to install a new Fronius PV inverter and after a Software update on it i was not able to show or fiind it on the VRM. So i googled and found a Forum where somebody told he deleted some files and after this it worked. I did the same but after this al my settings and serialbattery manager were gone.

so i tried to install the new version and it worked fine, but i was not able to open the utilitys tab to change the settings to the values i need.
it says **permission denied**. I already reinstalled it several times but the problem stays the same.

![image](https://github.com/Louisvdw/dbus-serialbattery/assets/148790318/0d3b6520-ed04-43f6-9843-a4f49cd9738b)

i hope somebody here can help me

thanks

**Comment by mr-manuel** [comment]
Please read the documentation. If you still need help open a support request under issues.

**Reply by Martinmaker92** [reply]
Thanks for your reply, you mean the how to install on github?
I read it but i am unsure whats to do.

Should i just open a support request or can you send my request there?

**Reply by mr-manuel** [reply]
You have to open one, since you did not post all needed informations.
